### DISCUSSION TITLE
plotman reload config?

### DISCUSSION THREAD

**Discussion by nodwan** [ericaltendorf/plotman#553] [Q&A]
my question is, if the system running, and there was a change in config file, for example, max plot‘s? how reload the config file?

**Comment by nodwan** [comment]
So, i was read more the wiki, i‘dont found a readme, how to change, for example jobs in inactive mode. is there some menue or witch keys are to press … ???

**Comment by ronerio** [comment]
I have the same question because i changed the config and no new plots started. I think we need start the plotman again.

**Comment by cyperbg** [comment]
Quit Plotman interactive with q and then start it again. That reloads the config.

**Comment by Eammon** [comment]
If you go to the terminal that you pressed Plotman Plot in you can hit  ctrl+c to cancel the process then press Plotman plot again. To test the new config setting have come into affect when new plots start you can use plot man details plotId and will show you the details of the new plot. I do this every time to ensure it has changed

**Reply by ronerio** [reply]
Thanks, it worked!
